functions/access.js

```diff
@@ -136,7 +136,7 @@ const createToken = async (venueId, uid, password, email, code) => {
 };
 
 exports.checkAccess = functions.https.onCall(async (data, context) => {
```

Comment:
Should we be using our generic `checkAuth` helper function at the start of this (and other endpoints in this file)?

- https://github.com/sparkletown/sparkle/blob/staging/functions/auth.js#L4-L12

Reply:
Yeah I'll add.

Reply:
Thanks :)